etc/kayobe/ansible/configure-vxlan.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Configure VXLAN
-  hosts: controllers,compute,seed,storage
+  hosts: "{{ configure_vxlan_playbook_hosts | default('seed,compute,controllers,storage') }}"
```

Comment:
Why not add infra-vms to the default?

Reply:
I wanted to keep the playbook as general as possible. I assumed that in a normal deployment we might want to treat infrastructure VMs differently and only configure the other hosts.

Reply:
This playbook is just for multinode, so don't need to be too careful.

Reply:
I've reverted the change and just added the group